config/bootstrap.rb

```diff
@@ -32,3 +32,10 @@ def perform
     add_transaction_tracer :perform, :category => :task
   end
 end
+
+require 'routemaster/mixins/log'
+include Routemaster::Mixins::Log
```

Comment:
I'm not sure why you'd need to do either patching or extending? Actually I'm not sure why you want to add anything to the bootstrap?
That's why I wrote this:

> Instead of forcing this in the bootstrap, you could for instance choose the path of lazyness and `warn` on the first log call.